AEPAssurance/Source/AssuranceSessionOrchestrator.swift

```diff
@@ -24,68 +40,114 @@ class AssuranceSessionOrchestrator: AssurancePresentationDelegate {
 
```

Comment:
QQ: Is this session variable thread safe?  It's flipped to nil by terminate, but it's used after guard by other functions.  Is there ever a place where queueEvent or similar could be called on a different threat than terminate gets called on, causing a runtime error?

Reply:
Yeap session variable is accessed from either
- Listener thread (queue the events)
- Extension Registration thread (starting session on boot up)
- UI Main thread (handling the responses from pincode and start session)

I need make the `session` thread safe.. Will do it in the next PR (Since it break a few tests )